packages/compiler/lib/reflection.cadl

```diff
@@ -0,0 +1,12 @@
+namespace Cadl.Reflection;
+
+model ModelProperty {}
+model Interface {}
+model Enum {}
+model EnumMember {}
+model TemplateParameter {}
+model Namespace {}
+model Operation {}
+model Tuple {}
```

Comment:
Do we need Tuple? I don't think you can decorate a tuple directly can you?

Reply:
Yeah don't think we need tuple